### PR TITLE
Added buttons add & remove

### PR DESCRIPTION
**Pull request by okhaimie-dev**
## Screenshots (if appropriate):
image

## How Has This Been Tested?
Some bug in the positions index. index > 2 results in an error ( ParticipantRow - line 29 ), so I set the index to 0 for now until I fix.

### CODE REVIEW

**Comment by aahna-ashina** [comment]
@xPi2 Can you setup automatic Vercel preview deployments for this GitHub repo?  That would make UI changes like these much easier to review.

**Comment by xPi2** [comment]
> @xPi2 Can you setup automatic Vercel preview deployments for this GitHub repo? That would make UI changes like these much easier to review.

When the project is deployed

File: packages/web-app/components/create/AgreementCreationForm.tsx [hunk 1]
```diff
@@ -16,6 +16,20 @@ export const AgreementCreationForm = () => {
...
+		setParticipantRow([...participantRow, ParticipantRow]);
+	}
+
+	// handle click event to remove button.
+	const handleRemoveClick = (index: number) => {
+		const list = [...participantRow];
+		list.splice(index, 1);
+		setParticipantRow(list);
+	};
+
```
Comment (xPi2) [inline packages/web-app/components/create/AgreementCreationForm.tsx:32]: You should be updating positions, not creating a new array of react elements. You are not using this array for more than the length. When you add more than two elements here you create a mismatch between the length of array of positions that is used in PaticipantRow components and the number of ParticipantRow components your are rendering.

File: packages/web-app/components/create/AgreementCreationForm.tsx [hunk 2]
```diff
@@ -56,8 +70,42 @@ export const AgreementCreationForm = () => {
...
 				</div>
 				<div className="flex flex-col gap-2">
-					<ParticipantRow positions={positions} index={0} onChange={setPositions} />
-					<ParticipantRow positions={positions} index={1} onChange={setPositions} />
+					{participantRow.map((singleParticipantRow, index) => (
+
+						<div className="flex w-full space-x-2">
+							<div className="flex-1">
+								{/* TODO index > 2 = break code */}
+								<ParticipantRow positions={positions} index={0} onChange={setPositions} />
```
Comment (xPi2) [inline packages/web-app/components/create/AgreementCreationForm.tsx:78]: > When you add more than two elements here you create a mismatch between the length of array of positions that is used in PaticipantRow components and the number of ParticipantRow components your are rendering.

As mentioned above.

File: packages/web-app/components/create/AgreementCreationForm.tsx [hunk 3]
```diff
@@ -56,8 +70,42 @@ export const AgreementCreationForm = () => {
...
+										type="button"
+										onClick={() => handleRemoveClick(index)}
+										className="h-full flex items-center"
+									>
+										<MinusCircleIcon className="w-7 h-7 text-[#F87272]" />
+									</button>
+								)}
+							</div>
+							</div>
+						</div>
```
Comment (xPi2) [inline packages/web-app/components/create/AgreementCreationForm.tsx:106]: This should be abstracted to a component or included into ParticipantRow

**Review by xPi2 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**